app/parse.py

```diff
@@ -8,5 +13,46 @@
     duration: str
 
 
-def get_all_courses() -> list[Course]:
-    pass
+def parse_courses(course_elements: Tag) -> Course:
+    name = course_elements.select_one(".typography_landingH3__vTjok").text
+    short_description = course_elements.select_one(
+        ".typography_landingTextMain__Rc8BD.mb-32"
+    ).text.strip()
+    duration_elements = course_elements.select(
+        ".typography_landingTextMain__Rc8BD."
+        "ProfessionCardTags_regularTag__yTc6K"
```

Comment:
The CSS classes `.typography_landingTextMain__Rc8BD.ProfessionCardTags_regularTag__yTc6K` are hardcoded. This could lead to issues if the website's structure changes. Consider using a more flexible approach.